servicenow/icon_servicenow/triggers/incident_created/trigger.py

```diff
@@ -0,0 +1,68 @@
+import komand
+import time
+from .schema import IncidentCreatedInput, IncidentCreatedOutput, Input, Output, Component
+# Custom imports below
+
+
+class IncidentCreated(komand.Trigger):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='incident_created',
+                description=Component.DESCRIPTION,
+                input=IncidentCreatedInput(),
+                output=IncidentCreatedOutput())
+        self.found = {}
+        self.url = ""
+        self.method = "get"
+        self.query = ""
+
+    def initialize(self):
+        if self.query:
+            response = self.connection.request.make_request(
+                self.url, self.method, params=self.query)
+        else:
+            response = self.connection.request.make_request(self.url, self.method)
+
+        try:
+            results = response.get("resource").get("result")
+        except KeyError as e:
```

Comment:
```suggestion
            results = response["resource"]["result"]
        except KeyError as e:
```
Using .get() will not produce a KeyError - it will default to `None` if the key is not found, which will then cascade a NoneValue error to your second `.get()`. If you want to catch KeyError, use the code suggestion above which just checks the specific key directly.